conans/test/command/info_folders_test.py

```diff
@@ -153,6 +153,10 @@ def test_short_paths_home_set_acl(self):
             return
 
         folder = temp_folder(False)  # Creates a temporary folder in %HOME%\appdata\local\temp
+
+        out = subprocess.check_output("wmic logicaldisk %s get FileSystem" % os.path.splitdrive(folder)[0])
```

Comment:
@pvicente I have added this check, because otherwise the test errors in non NTFS drives.

Reply:
@memsharded so I presume that you have a non NTFS home directory in your machine.

Anyway it looks ok to me.

Reply:
Yees, yes, that was the issue. It was failing on my machine, because I run on a ramdisk (not NTFS). It might happen to other users running conan test suite.